appcelerator/titanium_mobile
# feat(android): add "imageTouchFeedback" to ImageView

**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-28447

**Summary:**
- Adds properties to `Ti.UI.ImageView`:
  * `imageTouchFeedback` (boolean)
  * `imageTouchFeedbackColor` (string)
- Applies ripple effect to foreground image when touched.
- Note that existing `touchFeedback` feature is applied to a view's background. If you assign a completely opaque image to an `ImageView`, such as a photo, then the background can't be seen. This new feature allows you to apply a ripple effect to the foreground image.

**Test:**
1. Build and run the below on Android.
2. Tap on the image and verify a light gray ripple is displayed.
3. Uncomment line: `imageTouchFeedbackColor: "yellow",`
4. Build and run on Android.
5. Tap on the image and verify a yellow ripple is displayed.

```javascript
const window = Ti.UI.createWindow();
window.add(Ti.UI.createImageView({
	image: "https://raw.githubusercontent.com/appcelerator/titanium_mobile/master/tests/Resources/ExifRotate90.jpg",
	imageTouchFeedback: true,
//	imageTouchFeedbackColor: "yellow",
	autorotate: true,
}));
window.open();
```
